with-html.d.ts

```diff
@@ -0,0 +1,5 @@
+import {Component} from 'react'
+import ReactMarkdownRoot from 'react-markdown'
+
+export default class ReactMarkdown extends Component<ReactMarkdownRoot.ReactMarkdownProps, {}> {}
```

Comment:
the actual `with-html.js` file has commonjs export, so this line should probably be:
```ts
declare class ReactMarkdown extends Component<ReactMarkdownRoot.ReactMarkdownProps, {}> {};
export = ReactMarkdown;
```
could/should probably also change the import to:
```ts
import ReactMarkdownRoot = require('react-markdown');
```

This way, the `.d.ts` will be compatible with `esModuleInterop` on or off.

Reply:
Thanks for the review @AviVahl :)